presenters/playlists_presenter.js

```diff
@@ -0,0 +1,16 @@
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../knexfile')[environment];
+const database = require('knex')(configuration);
+
+const PlaylistObject = require('../models/playlist_object.js');
+
+class PlaylistsPresenter {
+
+  async createResponse(allPlaylists){
+    return await Promise.all(allPlaylists.map(async (playlist) => {
```

Comment:
Great use of iteration here! Also, nice job figuring out that the async functions had to be wrapped in a Promise to work